src/UKHO.S100PermitService.StubService/StubData/PKS/response-Allinvalid-404.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we please rename this file as response-datanotfound-404.json. Please set copy always property on file.

Reply:
Resolved